Server/ReasnAPI/ReasnAPI/Services/ImageService.cs

```diff
@@ -1,16 +1,175 @@
 ﻿using ReasnAPI.Models.Database;
+using ReasnAPI.Models.DTOs;
+using System.Linq.Expressions;
+using ReasnAPI.Exceptions;
+using ReasnAPI.Models.Enums;
+using ReasnAPI.Mappers;
+
+
+namespace ReasnAPI.Services;
+public class ImageService(ReasnContext context)
+{
+    public IEnumerable<ImageDto> CreateImages(List<ImageDto> imageDtos)
+    {
+        var newImages = new List<Image>();
+
+        if (imageDtos[0].ObjectType == ObjectType.User && imageDtos.Count > 1)

```

Comment:
What if imageDtos is empty? It will throw index out of range then

Reply:
created guard statement